sh.exe.stackdump

```diff
@@ -0,0 +1,22 @@
+Exception: STATUS_STACK_OVERFLOW at rip=7FF8D7F2AAB7
```

Comment:
Hum, that's a rather odd situation!
We will try to replicate this issue and see how we can fix this in the devTools.

In the meanwhile, I deleted it and updated the `.gitignore`. This file will not bother us again!